JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -157,33 +152,26 @@ private fun TabWithSections(
 @Composable
 private fun TopicItem(topicKey: String, itemTitle: String) {
     val image = +imageResource(R.drawable.placeholder_1_1)
-    Padding(left = 16.dp, right = 16.dp) {
-        FlexRow(
-            crossAxisAlignment = CrossAxisAlignment.Center
-        ) {
-            inflexible {
-                Container(width = 56.dp, height = 56.dp) {
-                    Clip(RoundedCornerShape(4.dp)) {
-                        DrawImage(image)
-                    }
-                }
-            }
-            expanded(1f) {
-                Text(
-                    text = itemTitle,
-                    modifier = Spacing(16.dp),
-                    style = (+MaterialTheme.typography()).subtitle1)
-            }
-            inflexible {
-                val selected = isTopicSelected(topicKey)
-                SelectTopicButton(
-                    onSelected = {
-                        selectTopic(topicKey, !selected)
-                    },
-                    selected = selected
-                )
+    Row(
+        Spacing(left = 16.dp, right = 16.dp)
```

Comment:
named params are used elsewhere so maybe better to keep consistency here as well?
Row(modifier = Spacing...